packages/lastpass-cli/build.sh

```diff
@@ -3,11 +3,12 @@ TERMUX_PKG_DESCRIPTION="LastPass command line interface tool"
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="1.4.0"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_REVISION=2
 TERMUX_PKG_SRCURL=https://github.com/lastpass/lastpass-cli/archive/v$TERMUX_PKG_VERSION/lastpass-cli-$TERMUX_PKG_VERSION.tar.gz
 TERMUX_PKG_SHA256=99517545e4906230a1fb21a0c6c575210f127057ad5b50eeb3c8e9acb7edfca3
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libcurl, libxml2, openssl, pinentry"
+TERMUX_PKG_BUILD_DEPENDS="bash-completion"
```

Comment:
lastpass-cli CMake file searches for bash-completions, see https://github.com/lastpass/lastpass-cli/blob/v1.4.0/CMakeLists.txt#L23. Otherwise, bash completion file is not installed.

bash-completion is not required for the lpass program. So, it is not a direct dependency.

Reply:
Whack.
Oh well.

Reply:
Alright, yep it seems to be installing the completions.